bool newTransformer
---
int32 ExecStatus